src/components/Header.css

```diff
@@ -1,73 +1,53 @@
-header {
+.navbar-default {
     color: #000;
     min-height: 40px;
-    background: #fff;
-    height: 100px;
     border-bottom: 5px var(--brand-color) solid;
-    position: relative;
 }
 
 img.brand {
     max-width: 250px;
-    margin: 7px 10px 10px 0;
+    margin: 7px 50px 10px 0;
+    height: 100%;
 }
 
-.page-link {
-    font-size: 16px;
-    font-weight: bold;
-    text-transform: uppercase;
-    line-height: 60px;
-    margin-bottom: 0;
-    margin-right: 20px;
-    float: left;
-    color: var(--brand-color);
-}
-
-header a.page-link,
-header a.page-link:active,
-header a.page-link:focus {
-    color: var(--brand-color);
-}
-
-@media all and (max-width: 992px) {
-    header a.page-link,
-    header a.page-link:active,
-    header a.page-link:focus {
-        color: #FFF;
-    }
+.navbar-nav{
```

Comment:
minor: can you make sure we're using the same coding style everywhere? I think we should have spaces before the {

Reply:
fix this on https://github.com/kelimuttu/activate-rewrite/commit/c1b7c9f9af4cd3d2da77b56964164776df78598d